tests/macros/tastrepr.nim

```diff
@@ -0,0 +1,17 @@
+discard """
+output: '''
+var data = @["one", "two"]
+for (i, d) = in pairs(data):
```

Comment:
equals sign `=` should not be here

Reply:
Looks like gfor() needs to be update to handle tuple unpacking directly instead of relying on gsub()

Reply:
oh year, good catch. I was blind about that.

Reply:
@cooldome I thought so as well, but tuples can be recursive. I implement a different hack now.